src/admin/participants/handler.js

```diff
@@ -312,6 +318,11 @@ const updateAccount = async function (request, h) {
       ledgerAccountType: await request.server.methods.enums('ledgerAccountType')
     }
     await Participant.updateAccount(request.payload, request.params, enums)
+    if (request.payload.isActive !== undefined) {
+      const isActiveText = request.payload.isActive ? 'activated' : 'disabled'
```

Comment:
Should this be hardcoded or pout into the enum?

Reply:
same as previous